nemo/collections/asr/parts/numba/rnnt_loss/utils/rnnt_helper.py

```diff
@@ -112,7 +113,7 @@ def compute_costs_data(source: torch.Tensor, dest: torch.Tensor, fastemit_lambda
     if idx < length:
         copy_data_1d(source, dest, idx)
         dest[idx] *= -1.0
-        dest[idx] *= 1.0 + fastemit_lambda
+        dest[idx] *= numba.float32(1.0 + fastemit_lambda)
```

Comment:
is there a particular reason that previously you made the change of "llForward += llForward * self.fastemit_lambda_" but here retain the old way? Just curious